src/proposals/20231115/SparkEthereum_20231115.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { SparkPayloadEthereum, IEngine, Rates, EngineFlags, Address } from '../../SparkPayloadEthereum.sol';
+
+interface IForwarder {
+    function execute(address payload) external;
+}
+
+/**
+ * @title  November 15, 2023 Spark Ethereum Proposal -
+ * @author Phoenix Labs
+ * @dev
+ * Forum:
+ * Vote:
+ */
+contract SparkEthereum_20231115 is SparkPayloadEthereum {
+
+    using Address for address;
+
+    address public constant RETH             = 0xae78736Cd615f374D3085123A210448E74Fc6393;
+    address public constant WSTETH           = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    address public constant DAI              = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public constant WETH             = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public constant WBTC             = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+    address public constant WBTC_PRICE_FEED  = 0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43;
+    address public constant GNOSIS_FORWARDER = 0x44f993EAe9a420Df9ffa5263c55f6C8eF46c0340;
+    address public constant GNOSIS_PAYLOAD   = 0xF62849F9A0B5Bf2913b396098F7c7019b51A820a;
```

Comment:
Understand this is a placeholder, but it should be specified as such with a comment to be replaced.